example/index.ts

```diff
@@ -0,0 +1,57 @@
+import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
+
+interface UseGravityArgs {
+  moverMass: number;
+  attractorMass: number;
+  attractor: Vector<number>;
+}
+
+const applyForceForStep = (
+  config: UseGravityArgs,
+  mover: Entity,
+  entity: Entity
+) => {
+  const force = forceV({
+    ...config,
+    mover: mover.position
+  });
+
+  return applyForce({
+    force,
+    moverMass: config.moverMass,
+    acceleration: mover.acceleration,
+    velocity: mover.velocity,
+    position: mover.position
+  });
+};
+
+const run = (config: UseGravityArgs) => {
+  const { start, stop } = rAF();
+
+  start((timestamp, lastFrame, mover) => {
```

Comment:
I fell like `mover` could be part of the run clojure here instead of being so tied to `rAF`. 

Would love to get your thoughts on how you got on this route.

Reply:
So I ended up getting here when trying to think of how the :heck: I preserve state across frames. I agree that `run` is a better place for this – but at the time of writing `rAF` I didn't yet have `run` implemented, so it ended up living in there. I agree that keeping `rAF` isolated to being a scheduler is the better approach, and as I begin to write the `gravity` function (basically what `run` is), that state will move out of `rAF`.